lab_3_keywords_textrank/main.py

```diff
@@ -51,7 +52,12 @@ def _clean_and_tokenize(self, text: str) -> tuple[str, ...]:
             tuple[str, ...]
                 clean lowercase tokens
         """
-        pass
+        for i in text:
+            if i in self._punctuation:
+                text = text.replace(i, ' ')
+        splited_text = text.lower().split()
+        tokens = [token for token in splited_text]
```

Comment:
You don't need to generate list before tuple. Let's generate tuple immediately

Reply:
Like in `_remove_stop_words`

Reply:
fixed